### PR TITLE
top: Migrate Python drivers/libraries from main repo to micropython-lib.

### PR DESCRIPTION
**Pull request by jimmo**
Companion to https://github.com/micropython/micropython-lib/pull/525

Updates all board manifests to use `require()` instead of `include()`, and then removes the libraries from this repo.

(This won't pass CI until https://github.com/micropython/micropython-lib/pull/525 is merged and the submodule updated)

### CODE REVIEW

**Comment by jimmo** [comment]
Note: @garatronic the `GARATRONIC_PYBSTICK26_F411` board definition does something a bit non-standard that sort of worked by accident to end up with the `ssd1306` module frozen as `display.ssd1306`.

Now that `ssd1306.py` is in `micropython/display/ssd1306/ssd1306.py` this trick doesn't work.

So this will be a breaking change. Do we need to add a supported workaround for this?

File: ports/esp8266/boards/GENERIC_512K/manifest.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 module("_boot.py", opt=3)
 module("apa102.py", base_path="$(PORT_DIR)/modules", opt=3)
 module("port_diag.py", base_path="$(PORT_DIR)/modules", opt=3)
-module("ntptime.py", base_path="$(MPY_DIR)/extmod", opt=3)
-include("$(MPY_DIR)/drivers/dht")
-include("$(MPY_DIR)/drivers/onewire")
-include("$(MPY_DIR)/extmod/webrepl")
-include("$(MPY_DIR)/drivers/neopixel")
+require("ntptime.py")
```
Comment (dpgeorge) [inline ports/esp8266/boards/GENERIC_512K/manifest.py:4]: should this be `"ntptime"`?  does it work if you have the `.py`?  does it give an error or silently fail?

Comment (jimmo) [reply ports/esp8266/boards/GENERIC_512K/manifest.py]: Fixed.

**Comment by jimmo** [comment]
Rebased.

**Comment by dpgeorge** [comment]
> the `GARATRONIC_PYBSTICK26_F411` board definition does something a bit non-standard that sort of worked by accident to end up with the `ssd1306` module frozen as `display.ssd1306`.
> 
> Now that `ssd1306.py` is in `micropython/display/ssd1306/ssd1306.py` this trick doesn't work.
> 
> So this will be a breaking change. Do we need to add a supported workaround for this?

I don't think we should workaround this, this board should be brought inline with other ones.

**Comment by dpgeorge** [comment]
Thank you!

**Comment by garatronic** [comment]
Thanks Damien, du to the lack of STM32F411 are fair price, i do not build this board. 
Can you send me your postal address at [ ***@***.*** | ***@***.*** ] . I would like to send you a new pybstick that seems to work pretty well. 

Frederic 


De: "Damien George" ***@***.***> 
À: "micropython" ***@***.***> 
Cc: "Frédéric Pierson" ***@***.***>, "Mention" ***@***.***> 
Envoyé: Jeudi 8 Septembre 2022 05:05:51 
Objet: Re: [micropython/micropython] top: Migrate Python drivers/libraries from main repo to micropython-lib. (PR #9220) 





Thank you!

**Pull request merged**